exercise_0/characterCounter/CharacterCounter.h

```diff
@@ -0,0 +1,28 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+#pragma once
+#include <string>
+
+class CharacterCounter {
+
+public:
+    CharacterCounter() = default;
+    void checkCharacter(std::string& word);
+    void printCounts() const;
```

Comment:
Implementing prints and formatting is usually done by implementing `operator <<`. Check out this example: https://coliru.stacked-crooked.com/view?id=17454ddf4005320d